packages/editor/src/editor/sidebar/fields/InputFieldWithBrowser.tsx

```diff
@@ -19,27 +21,39 @@ export const InputFieldWithBrowser = ({
 }: InputFieldProps & { browsers: Array<BrowserType>; options?: TextBrowserFieldOptions }) => {
   const [open, setOpen] = useState(false);
   const inputRef = useRef<HTMLInputElement>(null);
+  const cmsQuickRef = useRef<HTMLDivElement>(null);
```

Comment:
Maybe you could rename this to something more specific, like "cmsQuickFixPopover".